pkg/alertmanager/alertmanager.go

```diff
@@ -368,17 +368,21 @@ func (am *Alertmanager) ApplyConfig(userID string, conf *config.Config, rawCfg s
 		return nil
 	}
 
-	muteTimes := make(map[string][]timeinterval.TimeInterval, len(conf.MuteTimeIntervals))
+	timeIntervals := make(map[string][]timeinterval.TimeInterval, len(conf.MuteTimeIntervals)+len(conf.TimeIntervals))
 	for _, ti := range conf.MuteTimeIntervals {
-		muteTimes[ti.Name] = ti.TimeIntervals
+		timeIntervals[ti.Name] = ti.TimeIntervals
+	}
+
+	for _, ti := range conf.TimeIntervals {
+		timeIntervals[ti.Name] = ti.TimeIntervals
```

Comment:
If the same name is specified in both `muteTimes` and `timeIntervals`, shall we overwrite using `timeIntervals` or merge them?
Btw if it is not hard it would be great to have a unit test.

Reply:
OK. Looks like this is the same logic from upstream alertmanager then we should be good. https://github.com/prometheus/alertmanager/pull/2779/files